system-test/spanner.ts

```diff
@@ -275,6 +275,47 @@ describe('Spanner', () => {
         });
       });
     }
+    function readUntypedData(column, value, dialect, callback) {
+      const id = generateName('id');
+      const insertData = {
+        Key: id,
+        [column]: value,
+      };
+
+      let table = googleSqlTable;
```

Comment:
This should also go into if block. `if (dialect === Spanner.GOOGLE_STANDARD_SQL)`

Reply:
We override on line 293 if it is postgres. This is the default value